### PR TITLE
feat: add Noble

### PR DESCRIPTION
**Pull request by johnletey**
A follow up to #26, with the review comments addressed!

### CODE REVIEW

**Review by besated — APPROVED** [review]
thanks for contributing!

File: assetlist.json [hunk 1]
```diff
@@ -435,6 +435,28 @@
...
+        {
+          "denom": "USDC",
+          "exponent": 6
+        }
+      ],
+      "images": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.png"
+      },
+      "coingecko_id": "usd-coin",
```
Comment (besated) [inline assetlist.json:459]: @johnletey the workflow seems to be complaining about the trailing comma here, can you remove it?

**Review by besated — CHANGES_REQUESTED** [review]

File: assetlist.json (hunk 1 above)
Comment (johnletey) [reply assetlist.json]: Done!

**Review by besated — APPROVED** [review]
thanks!

**Comment by besated** [comment]
@johnletey it seems to be complaining about the trailing comma

**Pull request merged**